config/factory.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  defaults: {
+    type: 'proc',
+    test: true,
+    disposable: true,
+    ipfsModule: require('ipfs'),
+    ipfsHttpModule: require('ipfs-http-client')
+  },
+  overrides: {
+    js: {},
+    go: {
+      args: ['--enable-pubsub-experiment'],
+      ipfsBin: require('go-ipfs-dep').path()
+    }
+  }
+}
```

Comment:
This is the new `ipfsd-ctl` way of doing things where you provide a default and overrides. It works well, and will work even better when we add `rust` :sunglasses: 

However, it messes with our method of looping through the different types of IPFS nodes in our tests. Will have to think about this more.

Reply:
It also messes with the `TEST=****` environment variable technique :thinking: